chef/cookbooks/crowbar/templates/default/crowbar-delete-Member-role-before-upgrade.sh.erb

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#
+# This script deletes the Member role as part of the upgrade from Cloud 8 to
+# Cloud 9. This is necessary because the "Member" role was renamed to "member"
+# in Cloud 9 and the attempt to create the "member" role will fail if "Member"
+# exists already (Keystone is case-insensitive).
```

Comment:
Slight nitpick here, it's not keystone that is case-insensitive but the mysql backend. If it was up to keystone, we wouldn't have this problem.

Reply:
Why are we creating the member role in Crowbar? It is being created by 'keystone-manage bootstrap'. See

https://github.com/openstack/keystone/blob/stable/rocky/keystone/cmd/bootstrap.py#L112

Are we not using bootstrap?